book/src/cluster-driver-registrar.md

```diff
@@ -20,9 +20,9 @@ The CSI `cluster-driver-registrar` is a sidecar container that registers a CSI D
 CSI drivers that use one of the following Kubernetes features should use this sidecar container:
 
 * [Skip Attach](skip-attach.md)
-  * For drivers that don't support `ControllerPublishVolume`, this indicates to Kubernetes to skip the attach operation and eliminates the need to deploy the `external-attacher` sidecar (and in some cases, eliminates the need to deploy any "controller" components at all).
+  * For drivers that don't support [`ControllerPublishVolume`](https://github.com/container-storage-interface/spec/blob/master/spec.md#controllerpublishvolume), this indicates to Kubernetes to skip the attach operation and eliminates the need to deploy the `external-attacher` sidecar (and in some cases, eliminates the need to deploy any "controller" components at all).
 * [Pod Info on Mount](pod-info.md)
-  * This causes Kubernetes to pass metadata such as Pod name and namespace to the `NodePublish` call.
+  * This causes Kubernetes to pass metadata such as Pod name and namespace to the `NodePublishVolume` call.
```

Comment:
Do we want to document the exact information that's given in the cluster-driver-registrar [readme](https://github.com/kubernetes-csi/cluster-driver-registrar/pull/20#discussion_r250015457)?

Reply:
Will capture specific information in the "pod info on mount" feature page.